dizoo/atari/envs/atari_muzero_env.py

```diff
@@ -0,0 +1,202 @@
+            from gym.wrappers import Monitor
+            env = Monitor(env, directory=save_path, force=True, video_callable=video_callable, uid=uid)
+        return AtariWrapper(env, discount=self.cfg.discount, cvt_string=self.cfg.cvt_string)
+
+    def reset(self):
+        if not self._init_flag:
+            self._env = self._make_env()
+            self._observation_space = self._env.env.observation_space
+            self._action_space = self._env.env.action_space
+            self._reward_space = gym.spaces.Box(
+                low=self._env.env.reward_range[0], high=self._env.env.reward_range[1], shape=(1,), dtype=np.float32
+            )
+
+            self._init_flag = True
+        if hasattr(self, '_seed') and hasattr(self, '_dynamic_seed') and self._dynamic_seed:
+            np_seed = 100 * np.random.randint(1, 1000)
+            self._env.env.seed(self._seed + np_seed)
+        elif hasattr(self, '_seed'):
+            self._env.env.seed(self._seed)
+
+        obs = self._env.reset()
+        self.obs = to_ndarray(obs)
+        self._final_eval_reward = 0.
+
+        self.has_reset = True
+        self.agents = self.possible_agents[:]
+
+        self._agent_selector = agent_selector(self.agents)
+        self.agent_str = self._agent_selector.reset()
+        agent = self.agent_str
+
+        self.current_player_index = self.agents.index(agent)
+        obs = self.observe()
+        return BaseEnvTimestep(obs, None, None, None)
+
+    def observe(self):
+        observation = self.obs
+        action_mask = np.ones(self._action_space.n, 'int8')
+        return {'observation': observation, 'action_mask': action_mask}
+
+    def step(self, action):
+        current_agent = self.agent_str
+        self.current_player_index = self.agents.index(current_agent)
+        self.agent_str = self._agent_selector.next()  # next player to play
+        action = int(action)
+        obs, reward, done, info = self._env.step(action)
+        # self._env.render()
+        self._final_eval_reward += reward
+        self.obs = to_ndarray(obs)
+        self.reward = to_ndarray([reward])
+        observation = self.observe()
+        info = {'next player to play': self.agent_str}
+
+        if done:
+            info['final_eval_reward'] = self._final_eval_reward
+
+        return BaseEnvTimestep(observation, self.reward, done, info)
+
+    def legal_actions(self):
+        return np.arange(self._action_space.n)
+
+    @property
+    def observation_space(self) -> gym.spaces.Space:
+        return self._observation_space
+
+    @property
+    def action_space(self) -> gym.spaces.Space:
+        return self._action_space
+
+    @property
+    def reward_space(self) -> gym.spaces.Space:
+        return self._reward_space
+
+    def random_action(self):
+        action_list = self.legal_actions()
+        return np.random.choice(action_list)
+
+    def render(self, mode='human'):
+        self._env.render()
+
+    def human_to_action(self):
```

Comment:
this method is unnecessary

Reply:
may be used in test_atari_muzero_env.py, to be consistent with other board_games env